docker-compose.yml

```diff
@@ -37,6 +37,7 @@ services:
       - run:/var/run/postgresql
     labels:
       com.dnsdock.alias: instance.tba.docker
+    command: ["-c", "shared_preload_libraries=pg_stat_statements"]
```

Comment:
cc @pgiraud

Reply:
I've done the following on temboardUI https://github.com/dalibo/temboard/commit/77580c2a60882266ba124088806477948fa22856